python/src/trezorlib/cli/cosi.py

```diff
@@ -34,7 +34,7 @@ def cli() -> None:
 
 
 def cosi_warn(client: "TrezorClient") -> None:
-    if client.features.model == "1" and client.version < (1, 11, 2):
+    if client.features.internal_model == "T1B1" and client.version < (1, 11, 2):
```

Comment:
perhaps we should probably be using `client.model is models.TREZOR_ONE` and similar everywhere

Reply:
to make the code shorter, we could introduce `T1B1 = TREZOR_ONE` in models.py